mongo/cursor.go

```diff
@@ -186,10 +186,18 @@ func (c *Cursor) Close(ctx context.Context) error {
 func (c *Cursor) All(ctx context.Context, results interface{}) error {
 	resultsVal := reflect.ValueOf(results)
 	if resultsVal.Kind() != reflect.Ptr {
-		return errors.New("results argument must be a pointer to a slice")
+		return errors.New("results argument must be a pointer to a slice, but was a" + resultsVal.Kind().String())
```

Comment:
nit: Use `fmt.Errorf` rather than string concatenation:

```
return fmt.Errorf("results argument must be a pointer to a slice, but was a %s", resultsVal.Kind())
```

It makes for cleaner code and you don't have to worry about spacing. In this case, you're missing a space at the end of your error, so it would show up as `aint` rather than `a int`.

Reply:
done!